low-level/sspl-ll.spec

```diff
@@ -94,8 +94,9 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/resource_health_view /usr/bin/resource_health_view
     ln -sf $SSPL_DIR/low-level/sspl_bundle_generate /usr/bin/sspl_bundle_generate
     ln -sf $SSPL_DIR/low-level/manifest_support_bundle /usr/bin/manifest_support_bundle
-    ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/resource_map/lr2/
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/resource_map $SSPL_DIR/bin/resource_map
+    ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/solution/lr2/server/
```

Comment:
can you elaborate on these soft links based on new structure ?

Reply:
to use the framework module inside the solution directory without adding the python path we made this soft link. It will help to use the framework module.